public/index.html

```diff
@@ -107,9 +107,7 @@
   </head>
 
   <body>
-    <noscript>
-      You need to enable JavaScript to run this app.
-    </noscript>
+    <noscript> You need to enable JavaScript to run this app. </noscript>
```

Comment:
`<noscript>` is unknow before, which should be `inline`, and now current master is  `inline` too, why this changed.